static-analysis-plugin/plugin/src/test/groovy/com/novoda/test/TestAndroidProject.groovy

```diff
@@ -0,0 +1,62 @@
+package com.novoda.test
+
+class TestAndroidProject extends TestProject {
+    private static final Closure<String> TEMPLATE = { TestProject project ->
+        """
```

Comment:
But then we need to use some templating facility. I mean this will be in memory and visible only in one place - here. I'm open to other ideas tho.